rmw_opendds_cpp/src/rmw_publisher.cpp

```diff
@@ -102,9 +102,9 @@ rmw_create_publisher(
   const rmw_qos_profile_t * rmw_qos,
   const rmw_publisher_options_t * publisher_options)
 {
-  auto dds_node = OpenDDSNode::get_from(node);
+  auto dds_node = OpenDDSNode::from(node);
   if (!dds_node) {
-    return nullptr; // error set in get_from
+    return nullptr; // error set
```

Comment:
the comment no longer makes sense. Either remove it or update it.

Reply:
done